src/AppBundle/Resources/views/Pa/Index/clientEdit.html.twig

```diff
@@ -0,0 +1,37 @@
+{% extends 'AppBundle:Layouts:application.html.twig' %}
+
+{% block htmlTitle %}Edit client details{% endblock %}
+{% block pageTitle %}Edit client details{% endblock %}
```

Comment:
other PA's page are not translated, so I'd prefer to have Biggs doing that and keep keys/files naming consistent. Can't do that as none of the PA is translated. We were considering not using translation at a certain point, but not sure what it was agreed as more convenient. No preferences from my side @Biggsen